src/test/scala/cromwell/binding/ThreeStepImportNamespaceSpec.scala

```diff
@@ -0,0 +1,110 @@
+package cromwell.binding
+
+import org.scalatest.{FlatSpec, Matchers}
+
+class ThreeStepImportNamespaceSpec extends FlatSpec with Matchers {
+  val psTaskWdl = """
+    |task ps {
+    |  command {
+    |    ps
+    |  }
+    |  output {
+    |    File procs = stdout()
+    |  }
+    |}""".stripMargin
+
+  val cgrepTaskWdl = """
+    |task cgrep {
+    |  command {
+    |    grep '${pattern}' ${File in_file} | wc -l
+    |  }
+    |  output {
+    |    Int count = read_int(stdout())
+    |  }
+    |}""".stripMargin
+
+  val wcTaskWdl = """
+    |task wc {
+    |  command {
+    |    wc -l ${File in_file}
```

Comment:
I don't disagree.  I'm on the fence.  Seems like a good discussion for the Google Group.

Either way I'm okay with it being like this for this test if you are.